server/sonar-web/src/main/js/apps/projects/components/ProjectCard.js

```diff
@@ -74,6 +78,7 @@ export default class ProjectCard extends React.PureComponent {
               {project.name}
             </Link>
           </h2>
+          <TagsList tags={this.props.project && this.props.project.tags} hideWhenNoTags={true}/>
```

Comment:
you don't need to check `this.props.project`, it is already done before (line 49)